tastytaps/taps/models.py

```diff
@@ -30,3 +30,18 @@ class Price(models.Model):
 
     def __unicode__(self):
         return self.size
+
+
+class Taproom(models.Model):
+    name = models.CharField(max_length=200)
+    num_taps = models.PositiveSmallIntegerField(blank=True, null=True)
+    address = models.CharField(max_length=200, blank=True)
+
+    def __unicode__(self):
+        return self.name
+
+
+class Taps(models.Model):
+    beer = models.ForeignKey(Beer)
+    taproom = models.ForeignKey(Taproom, related_name='taps')
+    tap_number = models.PositiveSmallIntegerField(blank=True, null=True)
```

Comment:
I'm not sure if `tap_number` should be required. Let's say that you want to add this beer to the tap list but you don't know its number yet. I think, somehow, we should have some flexibility allowing the tap list creator to create the tap list and change the `tap_number` when they are done.